src/main/java/com/shop/controller/AdminController.java

```diff
@@ -0,0 +1,77 @@
+package com.shop.controller;
+
+import com.shop.model.Product;
+import com.shop.model.User;
+import com.shop.service.ProductService;
+import com.shop.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class AdminController {
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    ProductService productService;
+
+    @GetMapping("/admin/adminPage")
+    public String showAdminPage(){
+        return "adminPage";
+    }
+
+    @PostMapping("/admin/changeBanStatus")
+    public String changeBanStatus(@RequestParam("userToChange") String username)
+    {
```

Comment:
Maybe it would be better if we would do /admin/changebanStatus/{username} with PathVariable. This way admin would be able to change ban status without having to get to admin page.